.release-it.json

```diff
@@ -5,9 +5,6 @@
       "release": true
     }
   },
-  "src": {
```

Comment:
So what does this mean? Are we no longer using release-it? If we are shouldn't we still have a step here that performs the actual release?

Reply:
Everything remains the same. we run the `release-it` task and it commits and tags the release, just does not upload to bintray.

Reply:
So a human has to manually remember to do the upload after `release-it` is done?

Reply:
No. There is no upload task anymore. Once the release commit is made,  jitpack will build and distribute. see https://jitpack.io/#Skyscanner/backpack-android

Reply:
Okay so jitpack monitors our commits and builds/publishes when it sees a tagged commit on master?

Reply:
On the first request to that particular tag/commit, it builds and caches the artifacts.

Reply:
Hmm cool. Do we have something to ensure we have a changelog entry for each release?

Reply:
Will need to setup Unreleased and changelog similar to the backpack monorepo.